manifests/params.pp

```diff
@@ -70,7 +70,7 @@
   case $::osfamily {
     'RedHat', 'Suse', 'Archlinux': {
       $confdir              = '/etc/puppetdb/conf.d'
-      $embedded_subname     = 'file:/var/lib/puppetdb/db/db;hsqldb.tx=mvcc;sql.syntax_pgs=true'
+      $database_path        = '/var/lib/puppetdb/db/db'
```

Comment:
The embedded_subname was never a configurable parameter, so I've switched the focus here over to just the path - now we fill in the remaining parts of subname inside database_ini.pp in only one place (since it never should be different per OS anyway).